.gitignore

```diff
@@ -0,0 +1,37 @@
+# OSX
+#
+.DS_Store
```

Comment:
nah, please add here only files generated by the project that you don't want committed. `.DS_Store` should go into the *user* configuration, not the *project* configuration. In other terms, it's not your problem.